homeassistant/components/sms/notify.py

```diff
@@ -8,30 +8,36 @@
 from homeassistant.const import CONF_NAME, CONF_RECIPIENT, CONF_TARGET
 import homeassistant.helpers.config_validation as cv
 
-from .const import DOMAIN, GATEWAY, SMS_GATEWAY
+from .const import CONF_UNICODE, DOMAIN, GATEWAY, SMS_GATEWAY
 
 _LOGGER = logging.getLogger(__name__)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {vol.Required(CONF_RECIPIENT): cv.string, vol.Optional(CONF_NAME): cv.string}
+    {vol.Optional(CONF_RECIPIENT): cv.string, vol.Optional(CONF_NAME): cv.string}
 )
 
 
 def get_service(hass, config, discovery_info=None):
```

Comment:
We don't need this function if we implement `async_get_service`.

Reply:
Done